### DISCUSSION TITLE
[Feature] Autoback on selected network

### DISCUSSION THREAD

**Discussion by dfajmon** [immich-app/immich#13287] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a problem where I have enabled background backup. When out of home I get a lot of notifications `backup failed` on foreign network or on mobile data.

I would like to have a feature to be able to do background backup only on specific network. The notifications are getting annoying. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4176
